cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[tool.poetry]
+name = "{{ cookiecutter.project_name }}"
+version = "{{ cookiecutter.version }}"  # <<COOKIETEMPLE_FORCE_BUMP>>
+description = "{{ cookiecutter.project_short_description }}"
+authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
+license = "{{ cookiecutter.license }}"
+readme = "README.rst"
+homepage = "https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}"
+repository = "https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}"
+documentation = "https://{{ cookiecutter.project_slug }}.readthedocs.io"
+{% if cookiecutter.project_slug != cookiecutter.project_slug_no_hyphen -%}
```

Comment:
I am not sure why I did not delete this if statement, but there might be a reason?